ingestion-edge/ingestion_edge/disk_queue.py

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+from .conf import QUEUE_FILE
+from .conf import QUEUE_POSITION_FILE
+from flask import Blueprint
+from google.cloud.pubsub import PublisherClient
+from os import remove
+from os.path import exists
+from threading import Lock
+from typing import Tuple
+import ujson
+
+disk_queue = Blueprint("disk_queue", __name__)
+client = PublisherClient()
+read_lock = Lock()
+write_lock = Lock()
+
+
+def write(**kwargs):
```

Comment:
I think it would be more appropriate to take a single argument, instead of `**kwargs`. Semantically, this is taking a single item to write, not a set of key/value pairs.